### PR TITLE
Nga complete

### PR DESCRIPTION
**Pull request by borjasalguero**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b04e034d6da88bcf4df55ea17ec9ab02a9b190e3

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=498a37ee0c94fafcd1fbeda36b01a35945ab36df

File: apps/communications/contacts/js/views/ice_settings.js [hunk 1]
```diff
@@ -187,33 +183,11 @@ var contacts = window.contacts || {};
...
-
-    if (window.Search && Search.isInSearchMode()) {
-      Search.exitSearchMode();
-    }
-
-    contacts.Settings.navigation.back(() => {
-      hasICESet && contacts.List.toggleICEGroup(true);
-    });
-  }
+  // function resetIceGroupStates() {
```
Comment (arcturus) [inline apps/communications/contacts/js/views/ice_settings.js]: Dead code

File: apps/communications/contacts/views/settings/js/settings_controller.js [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+
+  // Initialise the settings controller events
+  function init() {
+
+    // TODO rename (includes outlook and gmail too) and delete FB part
+    fbLoader.load();
+
+
+    window.addEventListener('close-ui', function() {
+      window.history.go(-(window.history.length-1));
```
Comment (arcturus) [inline apps/communications/contacts/views/settings/js/settings_controller.js]: We need a huge comment explaning why we have to do this.

File: apps/communications/contacts/js/views/ice_settings.js (hunk 1 above)
Comment (borjasalguero) [inline apps/communications/contacts/js/views/ice_settings.js]: +1

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0d5e62cda3d92802ec18aebb3b301a17d2da9ec7

**Pull request merged**